src/styles/02-molecules/edition-card/_edition-card.scss

```diff
@@ -54,23 +54,24 @@
   }
 
   &__card-info-link {
-    @include button--filled;
-
-    flex: 1 1 100%;
-    &:not(:last-child) {
-      @include space-inline-s;
-    }
+    margin: auto; 
 
     @include breakpoint($medium) {
-      flex: 1 1 100%;
 
       &:not(:last-child) {
-        @include space-inline-none;
         @include space-stack-s;
       }
     }
   }
 
+  &__card-button-link {
+    @include button--filled;
+  }
+
+  &__more-link {
```

Comment:
For some reason I'm not seeing a `@space-inline-xs` or whatever it is between the icon and the "Download" string.

This link should also be black according to the specs, although I don't know why. We should reach out to Ellen tomorrow when she's in about updating the documentation for these cards (buttons are also getting reverted back to blue): http://themetronome.co/components/mc-14-edition-card/?fresh=true

Reply:
If the canonical version here should be blue, we should have it blue here ... but the button is canonically black as of right now (https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/button--button-filled-icon) and if there's been a change about that I'd rather file a new ticket rather than deal with updating all buttons in this ticket.

Reply:
Sorry for getting this muddled about buttons. Yes, I agree with you, if buttons need to be updated we should file a ticket and designs should be updated first.

As for the link, I think you should fix the spacing but I'm fine with leaving it stylistically in every other respect to our default links. While I think what Ellen's done here looks nice I'm not certain the logic for why she's made this link different from all other links on the site.

Reply:
ah it's more complicated than this - this is the button-like-link which probably ties into https://github.com/NYPL/nypl-design-system/issues/122.  Anyway, a ball of issues that need to be handled on their own.